Sources/MuxAVPlayerSDK/PublicAPI/Extensions/AVPlayerViewController+Mux.swift

```diff
@@ -59,23 +61,30 @@ extension AVPlayerViewController {
     /// to play your Mux Asset as well as monitor and report
     /// back it's playback performance.
     /// - Parameters:
-    ///   - publicPlaybackID: playback ID of the Mux Asset
+    ///   - playbackID: playback ID of the Mux Asset
     ///   you'd like to play
     ///   - customDomain: custom playback domain, custom domains
     ///   need to be configured as described [here](https://docs.mux.com/guides/video/use-a-custom-domain-for-streaming#use-your-own-domain-for-delivering-videos-and-images) first
-    convenience init(publicPlaybackID: String, customDomain: URL) {
+    convenience init(
+        playbackID: String,
+        customDomain: URL
```

Comment:
nitpick(blocking): I have a somewhat strong S1 here that `customDomain` should be a string. The reasons:
- custom (video) domains _**must**_ be `https` to be valid for our CNAME usage.
- custom (video) domains _**must**_ conform to the following format: `stream.{custom-domain}/{playback-id}.m3u8` as applied to Mux Video playlist GET requests
- When discussing custom (video) domains in our docs, the term doesn't refer to the entire hostname, but only the part that is replacing "mux.com". See, e.g.: https://docs.mux.com/guides/video/use-a-custom-domain-for-streaming

Reply:
The guide you've linked to mentions this

> For delivery, a custom domain allows you to play videos or deliver images from your domain rather than stream.mux.com or images.mux.com. Use your own domain for delivery, such as media.mycustomdomain.com

which means that e.g. `media.mycustomdomain.com` is a valid format?

Reply:
nope! from that guide:
> Video Playback URL: [https://stream.media.example.com/{playback_id}.m3u8](https://stream.media.exmaple.com/%7Bplayback_id%7D.m3u8)

Reply:
whoops that's from the blog post (https://www.mux.com/blog/introducing-custom-domains) but the same holds

Reply:
Indeed the blog post has more detail